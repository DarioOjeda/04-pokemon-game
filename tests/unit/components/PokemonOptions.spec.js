import { shallowMount } from "@vue/test-utils";
import PokemonOptions from "@/components/PokemonOptions";

import { pokemonsMock } from "../mocks/pokemons.mock";

describe("PokemonOptions Component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(PokemonOptions, {
      props: {
        pokemonsMock
      }
    });
  });

  test("debe hacer match con el snapshot", () => {
    console.log(wrapper.html());

    // expect(wrapper.html()).toMatchInlineSnapshot(`
    //   <div class="options-container" pokemonsmock="[object Object],[object Object],[object Object],[object Object]">
    //     <ul></ul>
    //   </div>
    // `);



    expect(wrapper.html()).toMatchSnapshot()
  });
});
