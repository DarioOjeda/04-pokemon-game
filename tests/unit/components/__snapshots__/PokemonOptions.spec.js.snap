// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonOptions Component debe hacer match con el snapshot 1`] = `
<div class="options-container" pokemonsmock="[object Object],[object Object],[object Object],[object Object]">
  <ul></ul>
</div>
`;
